$MirOS: ports/misc/mc/patches/patch-src_ext_h,v 1.1.7.1 2005/03/18 15:50:27 tg Exp $
--- src/ext.h.orig	Sun Aug 29 16:48:26 2004
+++ src/ext.h	Mon Feb 20 18:32:46 2006
@@ -8,7 +8,7 @@ int regex_command (const char *filename,
  */
 void flush_extension_file (void);
 
-#define MC_USER_EXT ".mc/bindings"
+#define MC_USER_EXT ".etc/mc/bindings"
 #define MC_LIB_EXT  "mc.ext"
 
 #endif				/* !__EXT_H */
